### DISCUSSION TITLE
Is 12.1 not coming to F-Droid?

### DISCUSSION THREAD

**Discussion by silentTeee** [libre-tube/LibreTube#3308] [Q&A] [answered]
As title says, I know that it usually takes 4-5 days for F-Droid to approve a release, but this time frame seemed oddly long. Was wondering if there was something on the GitHub release pipeline end, or if the team is thinking of waiting for another release instead?

**Comment by Bnyro** [comment, marked as answer]
In theory F-Droid should update the app on their repo automatically, I've just checked whether the release has been done correctly in order for F-Droid to detect it, everything looks fine so far.
So I guess F-Droid is just slow atm,

**Reply by silentTeee** [reply]
I see, okay no worries, thanks for the prompt reply!
